tensorcircuit/applications/optimization.py

```diff
@@ -318,9 +330,10 @@ def QUBO_QAOA_cvar(
     :param maxiter: The maximum number of iterations for the optimization. Default is 1000.
     :return: The optimized parameters for the ansatz circuit.
     """
+    tf.config.run_functions_eagerly(True)
```

Comment:
I dont think this is necessary for tf 2.0?

Reply:
exactly, this is because you impl is not jittable as I suggested below. By enabling this, the jit is closed. so no more no less, jit is still not enabled

Reply:
I see. Will fix it very soon.